mapfunction.rb

```diff
@@ -0,0 +1,7 @@
+#!/urs/bin/ruby -w
+
+arr = Array[1, 2, 3, 4, 5]
+
+arr = arr.map(){ |num| num = num*2; }
```

Comment:
1. Remove `()
2. Add space before `{` and around operators
3. Remove `;`
4. Is there a need to use `num = num * 2`?